modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/js/Tabs.js

```diff
@@ -63,6 +68,8 @@ export default function ({
 			display(navReplacementsId);
 			setActiveTabKeyValue(3);
 		}
+		setFirstRender(false);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Is this `eslint-disable` necessary or can `firstRender` be added to the list of dependencies without breaking functionality?

Reply:
yes, `display()` function reads the `firstRender`state so `useEffect` with non-empty dependencies array warns us about this. I believe we can consider this case an exception while we get the proper solution in the FDS side.

Reply:
I've not tried to add `firstRender`to the dependencies because we'd run the effect twice at least, as we're changing the state inside the effect. This is basically a compromise solution as the need to distinguish the first render case is due to the limitation we're overcoming. Said that, we may think aboit a different way of doing this w/o state

Reply:
We can, yea, those comments were made with urgency and hackiness in mind :)

Reply:
I think just go with whatever works, the goal is to remove this fix anyway. #techdebt